### ISSUE TITLE
NVDA mispronounces landmarks on web pages in SeaMonkey

### ISSUE THREAD

**Issue by jawhien** [nvaccess/nvda#11705]
Instead of Landmark, NVDA reports the "Section" or "text frame" on web pages in the SeaMonkey browser, and does not provide a name, unless explicitly specified, such as the "aria-label" attribute.

### Steps to reproduce:
1. Open any page in the SeaMonkey browser that has landmarks, for example: github.com.
2. Move through the content in any way, including the "D" key.

### Actual behavior:
NVDA does not correctly report landmarks and their names.

### Expected behavior:
NVDA should read landmarks and their names like other browsers.

#### NVDA installed/portable/running from source:
Installed/Portable.

#### NVDA version:
2020.2

#### Windows version:
Windows 7

#### Name and version of other software in use when reproducing the issue:
Seamonkey 2.53.4

#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
2019.2.1, landmarks are reported correctly.

#### If addons are disabled, is your problem still occuring?
Yes.

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Yes.

**Comment by leonardder** [comment]
I"m afraid this was regressed by https://github.com/nvaccess/nvda/pull/10110.
Can you still move through landmarks with the d quick navigation key?

**Comment by jawhien** [comment]
Yes, the "D" key moves focus correctly between landmarks.

**Comment by leonardder** [comment]
@MarcoZehe Could it be that Seamonkey uses a version of Gecko pre version 65?

**Comment by MarcoZehe** [comment]
@leonardder Yes, SeaMonkey uses Gecko 60.0, which hasn't seen security updates for ages. Even though releases are still coming, currently at version 2.53.4, they all use this old Gecko release, which even Thunderbird abandoned ages ago. Since this platform doesn't even officially get security updates any more, I would not use it in a daily usage environment at all any more. There's no way we will backport that landmark fix to this old Gecko version.

**Comment by leonardder** [comment]
@jawhien Would it be possible for you to reconsider your use of seamonkey based on https://github.com/nvaccess/nvda/issues/11705#issuecomment-701221993 ? It turns out that Firefox + Thunderbird is probably a much more safe way to go for you.

**Comment by jawhien** [comment]
@leonardder
Sorry, not currently. They are often unstable.

**Comment by leonardder** [comment]
Certainly not if you choose the ESR versions of both, particularly Firefox.

**Comment by jawhien** [comment]
@leonardder
I may not be right, but it is impossible to use new versions of thunderbird due to constant crashes.
In the end, if backward compatibility is not possible, we can close this issue. :(

**Comment by leonardder** [comment]
It should be possible to restore backwards compatibility, but I'm afraid 
that it could somehow break behavior for current versions.

**Comment by Adriani90** [comment]
@jawhien is this reproducible also in Seamonkey 2.53.16 with NVDA 2023.1?

Given @leonardder's comment above I tend to closing this issue as can't fix due to the risk of breaking current versions.

**Comment by Adriani90** [comment]
However, it seems there were security updates and enhancements updates backported up to Firefox and Thunderbird 102, so this might have been fixed now anyway.

**Comment by jawhien** [comment]
@Adriani90 
Hello. in version 2.53.16 and with nvda 2023.1 the problem is partially reproduced. some landmarks are declared normally. but some don't.

**Comment by Adriani90** [comment]
@jawhien how is this now with NVDA 2024.1 RC1 and Seamonkey 2.53.18.2? Is the issue still reproducible?

**Comment by jawhien** [comment]
@Adriani90
Частично. Некоторые теперь читаются корректно, footer и baner например. Точно не читается "main". Но есть другая большая проблема, nvda 2024 вызывает крах seamonkey очень часто.

**Comment by Adriani90** [comment]
@jawhien Are you still having this issue in Seamonkey 2.53.20?
https://www.seamonkey-project.org/releases/2.53.20

Did you already create a bug report in the Mozilla bug tracker?
https://bugzilla.mozilla.org/enter_bug.cgi?format=guided
